PyU4V/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 from .univmax_conn import U4VConn  # noqa: F401
 __title__ = 'pyu4v'
-__version__ = '3.1.1'
+__version__ = '3.1.2'
```

Comment:
Both #54 and #55 are updating their version to 3.1.2, should one commit be changed to 3.1.3 instead?